lib/entice/logic/Maps/map.ex

```diff
@@ -48,21 +50,8 @@ defmodule Entice.Logic.Area.Maps do
       def get_maps, do: @maps
     end
   end
-end
-
 
-defmodule Entice.Logic.Area.Maps.Map do
-  @moduledoc """
-  This macro puts all common map functions inside the map/area module that uses it
-  """
-  import Inflex
 
-  defmacro __using__(_) do
-    quote do
-      alias Entice.Utils.Geom.Coord
-      unquote(content(__CALLER__.module))
-    end
-  end
 
```

Comment:
reduce the copious whitespace a bit here ;) 2 empty lines between functions is enough ^^